node.go

```diff
@@ -14,9 +14,9 @@ type Node struct {
 	Host    Host
 	aliases []Host
 
-	cluster       *Cluster
-	pool          *Pool
-	refreshTicker *time.Ticker
+	cluster         *Cluster
+	pool            *Pool
+	refreshDoneChan chan bool
```

Comment:
Kinda the same but I slightly prefer the bool chan as it somewhat better communicates the `Done == true` fact. No strong feelings though.